src/components/View/Modulo/ModuloFormatiAlternativi.jsx

```diff
@@ -16,9 +16,9 @@ const ModuloFormatiAlternativi = ({ content }) => {
   return content.formato_alternativo_1 || content.formato_alternativo_2 ? (
     <div className="genericcard card card-teaser shadow p-4 mt-3 rounded">
       <div className="card-body">
-        <h5 className="card-title">
+        <h2 className="card-title h5">
```

Comment:
https://iosanita-staging.c3p0.redturtle.it/lista-blocchi/lista-ct/documento/modulo#:~:text=File%20principale-,Formati%20alternativi,-Quanto%20sono%20chiare